quadriga/src/main/java/edu/asu/diging/quadriga/api/v1/AddNetworkApiController.java

```diff
@@ -43,8 +57,21 @@
      * @return
      */
     @ResponseBody
-    @RequestMapping(value = "/api/v1/network/add", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE)
-    public HttpStatus processJson(@RequestBody Quadruple quadruple) {
+    @RequestMapping(value = "/api/v1/collection/{collectionId}/network/add", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE)
+    public HttpStatus processJson(@RequestBody Quadruple quadruple, @PathVariable String collectionId) {
+        
+        // Every time a new network is submitted, the triple in that network has to be added as the
+        // default MappedTripleGroup for given collectionId
+        MappedTripleGroup mappedTripleGroup;
+        try {
+            mappedTripleGroup = mappedTripleGroupService.getMappedTripleGroup(collectionId, MappedTripleType.DEFAULT_MAPPING);
+            if(mappedTripleGroup == null) {
+                return HttpStatus.NOT_FOUND;
+            }
+        } catch(InvalidObjectIdException | CollectionNotFoundException e)  {
+            logger.error(e.getMessage());
```

Comment:
you're swallowing the stacktrace, which makes it hard to impossible to figure out what went wrong. You have to pass the exception.

Reply:
The logger.error() method won't accept an Object of type Exception, I tried
So I had added ExceptionUtils.getStackTrace which helps maintain stack trace but in one of the PR comments you asked me to remove that
So I added e.getMessage()
I am not sure what exactly to do here
Is there any example I could check?